contributing/code/reproducer.rst

```diff
@@ -65,8 +65,9 @@ to a route definition. Then, after creating your project:
    of controllers, actions, etc. as in your original application.
 #. Create a small controller and add your routing definition that shows the bug.
 #. Don't create or modify any other file.
-#. Execute ``composer require symfony/web-server-bundle`` and use the ``server:run``
-   command to browse to the new route and see if the bug appears or not.
+#. Install the :doc:`local web server </setup/symfony_server>` provided by Symfony
+   and use the ``symfony server:start`` command to browse to the new route and
```

Comment:
`symfony serve`?

Reply:
„serve“ is just an alias. „server:start“ is fine 👌🏻

Reply:
`server:start` is also an alias :D

Reply:
Oh really ? 😊

Reply:
Both look equally correct to me ... so, to not waste much time discussing about this ... let's merge "as is" 😄